project/AppDependencies.scala

```diff
@@ -2,15 +2,15 @@ import sbt._
 
 object AppDependencies {
 
-  val bootstrapVersion = "9.1.0"
-  val mongoPlayVersion = "2.2.0"
+  val bootstrapVersion = "9.6.0"
+  val mongoPlayVersion = "2.3.0"
```

Comment:
In the meantime, mongo-play version 2.4.0 was released.
You could upgrade it, or leave as it is. It should not make a huge difference.